include_core/omrport.h

```diff
@@ -212,6 +212,7 @@
 #define OMRPORT_VMEM_PAGE_FLAG_FIXED 0x2
 #define OMRPORT_VMEM_PAGE_FLAG_PAGEABLE 0x4
 #define OMRPORT_VMEM_PAGE_FLAG_SUPERPAGE_ANY 0x8
+#define OMRPORT_VMEM_PAGE_FLAG_PAGEABLE_PREFERABLE 0x10
```

Comment:
I am asking again: Should mask in line below `OMRPORT_VMEM_PAGE_FLAG_TYPE_MASK` to be updated?! I understand that proposed flag is for ZOS and mask is for OSX but we should be consistent

Reply:
I don't see any reason why it shouldn't be updated. I've updated it to be consistent. 
See: 5c64063a8dd430ad91ac17b27f03616ebd21be41